From: Daniel Baker <dbaker@mozilla.com>
Date: Tue, 27 Aug 2024 19:24:38 -0600
Subject: (tmp-cherry-pick) Revert "Remove iOS Debug (simulator) from LKGR
 bots." (1ac162ee20)

This reverts commit 4bded9601bfb38f2ef67574554c12370dca4708f.

Reason for revert: Fixed with https://webrtc-review.googlesource.com/c/src/+/357640

Original change's description:
> Remove iOS Debug (simulator) from LKGR bots.
>
> Temporarily skip while the bot gets fixed.
>
> Bug: chromium:353975341
> Change-Id: Ib42c18e929547c7abc58f2878c79f00f87001cae
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/357540
> Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
> Auto-Submit: Mirko Bonadei <mbonadei@webrtc.org>
> Reviewed-by: Artem Titov <titovartem@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#42656}

Bug: chromium:353975341
Change-Id: I184c9e597a28fff3ae052a07d8e6f17cc2251188
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/357661
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Commit-Queue: Jeremy Leconte <jleconte@google.com>
Reviewed-by: Artem Titov <titovartem@webrtc.org>
Cr-Commit-Position: refs/heads/main@{#42664}
---
 infra/config/config.star        | 4 +---
 infra/config/cr-buildbucket.cfg | 1 +
 infra/config/luci-notify.cfg    | 5 +++++
 infra/config/project.cfg        | 2 +-
 4 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/infra/config/config.star b/infra/config/config.star
index 8a42399d9c..8955686887 100755
--- a/infra/config/config.star
+++ b/infra/config/config.star
@@ -52,9 +52,7 @@ def os_from_name(name):
 # Add names of builders to remove from LKGR finder to this list. This is
 # useful when a failure can be safely ignored while fixing it without
 # blocking the LKGR finder on it.
-skipped_lkgr_bots = [
-    "iOS Debug (simulator)",
-]
+skipped_lkgr_bots = []
 
 # Use LUCI Scheduler BBv2 names and add Scheduler realms configs.
 lucicfg.enable_experiment("crbug.com/1182002")
diff --git a/infra/config/cr-buildbucket.cfg b/infra/config/cr-buildbucket.cfg
index 9edb6a2cc6..456fe1fac0 100644
--- a/infra/config/cr-buildbucket.cfg
+++ b/infra/config/cr-buildbucket.cfg
@@ -1903,6 +1903,7 @@ buckets {
         '          "Win64 Debug (Clang)",'
         '          "Win64 Release (Clang)",'
         '          "iOS API Framework Builder",'
+        '          "iOS Debug (simulator)",'
         '          "iOS64 Debug",'
         '          "iOS64 Release"'
         '        ]'
diff --git a/infra/config/luci-notify.cfg b/infra/config/luci-notify.cfg
index e2835fb848..53a6c5eb3b 100644
--- a/infra/config/luci-notify.cfg
+++ b/infra/config/luci-notify.cfg
@@ -943,6 +943,11 @@ notifiers {
     name: "iOS Debug (simulator)"
     repository: "https://webrtc.googlesource.com/src"
   }
+  tree_closers {
+    tree_status_host: "webrtc-status.appspot.com"
+    failed_step_regexp: "bot_update|compile|gclient runhooks|runhooks|update|extract build|cleanup_temp|taskkill|compile|gn"
+    failed_step_regexp_exclude: ".*\\(experimental\\).*"
+  }
 }
 notifiers {
   notifications {
diff --git a/infra/config/project.cfg b/infra/config/project.cfg
index 8b377430f3..7ee089eda0 100644
--- a/infra/config/project.cfg
+++ b/infra/config/project.cfg
@@ -7,7 +7,7 @@
 name: "webrtc"
 access: "group:all"
 lucicfg {
-  version: "1.43.9"
+  version: "1.43.8"
   package_dir: "."
   config_dir: "."
   entry_point: "config.star"
